### PR TITLE
Add JavaScript/Node.js usage example

### PR DESCRIPTION
**Pull request by hugoabonizio**
Following #13, this PR adds a JavaScript example to call the API.

### CODE REVIEW

**Comment by rodrigo-f-nogueira** [comment]
Thanks for adding this! Since we are here, could you also add a link to this example in the README.md?

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ pip install maritalk
 
 # Exemplo de Uso
 
-Mostramos abaixo um exemplo simples de uso em Python.
+Mostramos abaixo um exemplo simples de uso em Python. Na [pasta exemplos](./examples/api/) existem mais códigos mostrando como chamar a API.
```
Comment (rodrigo-f-nogueira) [inline README.md:21]:
```suggestion
Mostramos abaixo um exemplo simples de uso em Python. Na [pasta exemplos](./examples/api/) existem mais códigos mostrando como chamar a API usando JavaScript/Node.js.
```

**Review by rodrigo-f-nogueira — APPROVED** [review]
Thanks for adding this!

File: README.md (hunk 1 above)
Comment (rodrigo-f-nogueira) [reply README.md:21]: If you prefer to be more generic in the description, feel free to ignore my suggestion...

Comment (hugoabonizio) [reply README.md:21]: I think we should maintain it more generic since we may add more examples in the future.

**Pull request merged**